examples/js/app.tsx

```diff
@@ -82,15 +82,9 @@ function ProductItem({ hit }: ProductItemProps) {
         <img src={hit.image} alt={hit.name} width="20" height="20" />
       </div>
 
-      <div
-        className="aa-ItemTitle"
-        dangerouslySetInnerHTML={{
-          __html: highlightHit<ProductHit>({
-            hit,
-            attribute: 'name',
-          }),
-        }}
-      />
+      <div className="aa-ItemTitle">
+        {highlightHit<ProductHit>({ hit, attribute: 'name' })}
```

Comment:
looking at this usage, how does highlightHit know what the createElement we pass to autocomplete is

Reply:
`highlightHit` accepts a `createElement` param that defaults to Preact's implementation: https://github.com/algolia/autocomplete.js/pull/399/files#diff-8c6073f7006317668cfb4ae052411c4fb3f0ac2f121f8fb923a2051054d9b402R26

(If we want to share this highlighting package, we shouldn't give it a default.)

Reply:
makes sense!